web/fido_app/static/scripts/register.js

```diff
@@ -192,6 +205,13 @@ const postNewAssertionToServer = async (credentialDataForServer, csrfToken) => {
 
 
 // Bind event listeners
+if (password) {
+  password.addEventListener("blur", checkPasswordLength);
+}
+else {
+  console.warn("\"Password\" button not found on page. Skipping event listener binding.");
```

Comment:
This is a very minor thing and an existing issue from past versions that we overlooked, but technically both this warning and the confirm password warning shouldn't say "button," since, well, they're not buttons.